misc/make_changelog.py

```diff
@@ -56,6 +59,7 @@ def format(c):
         s = s[r + 1:]
         s = s.strip()
     for tag in tags:
+        assert tag.lower() in details, f'Tag {tag} undefined in details'
```

Comment:
What if we accidentally add a undefined tag? Let's make this just warning for now.

Reply:
I would say it's better for everyone to follow the rules and use defined tags only. If a new flag is introduced, we can easily include that in https://github.com/taichi-dev/taichi/blob/a269888b49ae98f258d3f812774e4b2f8adb0ef1/misc/make_changelog.py#L23-L44